src/shared/lib/timers/model/TimerBase.ts

```diff
@@ -0,0 +1,67 @@
+import { AppState, NativeEventSubscription } from 'react-native';
+
+let backgroundMoveTime: number | null = null;
```

Comment:
Please make it a property of the class as we could have two timer instances at the same time